sport/app/football/views/wallchart/knockoutMatch.scala.html

```diff
@@ -25,9 +28,10 @@
             }
             @if(fm.isFixture){
                 <div class="football-match__date">
-                    <span class="football-match__kickoff">@fm.date.format(DateTimeFormatter.ofPattern("HH:mm"))</span>
+                    <span class="football-match__kickoff">@fm.date.format(DateTimeFormatter.ofPattern("HH:mm ").withZone(Edition(request).timezoneId))</span>
                     @fm.date.format(DateTimeFormatter.ofPattern("E dd MMMM"))
```

Comment:
I think you also need to also add the timezone here for the day of the week to be correct!

Reply:
Good catch!